### ISSUE TITLE
v3 va-segmented-progress-bar causes accessibility issue if an h1 is placed after it

### ISSUE THREAD

**Issue by rhasselle-oddball** [department-of-veterans-affairs/vets-design-system-documentation#2059]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

As part of a custom header prototype ([example 1](https://staging.va.gov/authorization-to-disclose-alternate), [example2](https://staging.va.gov/decision-reviews/appeals-testing/introduction)), We are experimenting with putting the h1 as the page title which is after the segmented progress bar. This causes an accessibility issue because it says that a h2, h3, h4 etc should not be before an h1. But we cannot remove or change the h2,h3,h4 from the progress bar.

## What I expected to happen

Be allowed to customize the segmented progress bar so that we can omit using a h1,h2,h3,h4 etc. perhaps use something else instead like a div.

## Reproducing
- Formation version:
- Device: (e.g. iPhone 8, Macbook)
- Browser: Chrome

Steps to reproduce:

Use a h1 after a va-segmented-progress-bar I notice accessibility issues from axe or something like HeadingsMap Chrome extension

## Urgency
How urgent is this request? Please select the approriate option below and/or provide details
- [x] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [ ] This bug is not blocking any work
- [ ] Other

**Comment by Andrew565** [comment]
It was decided that a new prop would be added - "useDiv" - that would force the segmented progress bar component to use a div instead of a header tag (h1, h2, etc). This "useDiv" prop would also cause the div that is used to look for an aria-describedby id to make sure that accessibility concerns are mitigated as best as possible.

**Comment by PSIEBENLIST** [comment]
Thanks for the attention and quick turn-around on this unusual request. Also thanks for the feedback on best practices for communicating/labeling tickets! Will keep in mind moving forward.

**Comment by Andrew565** [comment]
Will carry over due to needing to do more accessibility testing.
